From a0461649d3f88080cea8059110ecf4ba4864a770 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:58 -0400
Subject: [redhat] kabi: Adding symbol scsi_device_put (drivers/scsi/scsi.c)

Message-id: <4494b3aed9c7eef889f43d3b3c0492f2618c90b5.1410510773.git.poros@redhat.com>
Patchwork-id: 96101
O-Subject: [PATCH RHEL7.1 178/233] kabi: Adding symbol scsi_device_put (drivers/scsi/scsi.c)
Bugzilla: 1071099
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Release a reference to the scsi_device and decrements
    the use count of the underlying low level driver.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_device_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_device_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_device_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_put b/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_put
new file mode 100644
index 0000000..c5d02ac
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_put
@@ -0,0 +1,2 @@
+#1-
+0x24154c52 scsi_device_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_device_put b/redhat/kabi/kabi-module/kabi_s390x/scsi_device_put
new file mode 100644
index 0000000..5f324aa
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_device_put
@@ -0,0 +1,2 @@
+#1-
+0x416bbf68 scsi_device_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_put b/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_put
new file mode 100644
index 0000000..76e0361
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_put
@@ -0,0 +1,2 @@
+#1-
+0x68d95008 scsi_device_put vmlinux EXPORT_SYMBOL
-- 
1.7.1